pgtap/tsp/TSPeuclidean/tspEuclidean-rows-check.sql

```diff
@@ -0,0 +1,37 @@
+\i setup.sql
+
+SELECT plan(5);
+
+SET extra_float_digits = -3;
+
+-- Check whether the same set of rows is always returned
+
+PREPARE expectedOutput AS
+SELECT seq, node, cost::TEXT, agg_cost::TEXT FROM pgr_TSPeuclidean(
+    'SELECT id, st_X(the_geom) AS x, st_Y(the_geom)AS y
```

Comment:
Remove the TSPEuclidean test file, as you mentioned it sometimes passes, the thing is that the data is float, and in any case your fix does not work on this one